examples/bridge/src/application/actor.rs

```diff
@@ -64,14 +64,14 @@ impl<R: Rng + Spawner, H: Hasher, Si: Sink, St: Stream> Application<R, H, Si, St
         let (mut indexer_sender, mut indexer_receiver) = self.indexer.split();
         while let Some(message) = self.mailbox.next().await {
             match message {
-                Message::Genesis { response } => {
-                    // Use the digest of the genesis message as the initial
-                    // payload.
+                Message::Genesis { epoch, response } => {
+                    assert_eq!(epoch, 0, "epoch must be 0");
+                    // Use the digest of the genesis message as the initial payload.
                     self.hasher.update(GENESIS);
                     let digest = self.hasher.finalize();
                     let _ = response.send(digest);
                 }
-                Message::Propose { index, response } => {
+                Message::Propose { round, response } => {
```

Comment:
Do we want to assert this is also in epoch `0`?

I suppose that'll be handled for us by the consensus we've instantiated?